.github/CODEOWNERS

```diff
@@ -30,11 +30,8 @@
 /components/installation-telemetry @gitpod-io/engineering-security-infrastructure-and-delivery
 /components/kots-config-check @gitpod-io/engineering-security-infrastructure-and-delivery
 /install @gitpod-io/engineering-security-infrastructure-and-delivery
-/install/installer @gitpod-io/engineering-security-infrastructure-and-delivery
-# For testdata a single review from anyone who is allowed to review PRs is sufficent.
-/install/installer/cmd/testdata
-# Allow everyone to approve config changes
-/install/installer/pgk/config
```

Comment:
We no longer need to have these folders as explicitly "shared" anymore so I removed them